consensus/bor/heimdallapp/state_sync.go

```diff
@@ -0,0 +1,51 @@
+package heimdallapp
+
+import (
+	"context"
+	"encoding/json"
```

Comment:
Should we use json-iter instead?

Reply:
I am not sure, I used what I generally use to do json marshal/unmarshal, Also I checked other implementations of IHeimdallClient, We are using `encoding/json` there as well, Let me know if you want me to change/update.

Reply:
check other places - we started using go-iterator for quite a while

Reply:
Yes ser, On it